src/PockyBot.NET.Persistence/Repositories/PockyUserRepository.cs

```diff
@@ -38,5 +39,10 @@ public PockyUser AddOrUpdateUser(string userId, string username)
 
             return newPockyUser;
         }
+
+        public List<PockyUser> GetAllUsersWithPegs()
+        {
+            return _context.PockyUsers.Where(x => x.PegsGiven.Count > 0 || x.PegsReceived.Count > 0).ToList();
```

Comment:
.Any instead of Count > 0 ?

Reply:
I always forget that method exists lol. Will update